### PR TITLE
✨ Webpack config supports modern ES6+ features

### PR DESCRIPTION
**Pull request by fayzzzm**

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-const path = require("path");
-const { CleanWebpackPlugin } = require("clean-webpack-plugin");
-const { name } = require('./package.json')
+import * as path from "path";
+import { CleanWebpackPlugin } from "clean-webpack-plugin";
+
+import { name } from "./package.json";
```
Comment (fayzzzm) [inline webpack.config.babel.js:4]: Isn't it cool ?)

**Review by fayzzzm — COMMENTED** [review]
Need to write some optimization for our code and refactor several parts of the config. Can we create github-pages for testing the view of our component? All of our tests have been written around functions that are validating the input. We could create an overlay where we could install Vue and its' dependencies, and test our component view.
The things should be done by the next version release: 

- [ ] Should be provided performance tests for functions
- [ ] Refactor of core code and using meaningful names 
- [ ] Documentation editing
- [ ] Eslint config should be flexible and should control rules by APP_ENV (app environment).

@VoIdJMp, if you have any ideas, let me know and check it out (PR and the list on the top). 

PS: By finishing the review, don't merge it. I will do it by myself :))

File: webpack.config.babel.js (hunk 1 above)
Comment (VoIdJMp) [reply webpack.config.babel.js:4]: Ok, but I quickly wrote them for npm setup

**Comment by VoIdJMp** [comment]
You can customize github-pages yourself. Necessarily need to configure. I'll be busy writing the documentation.

**Comment by VoIdJMp** [comment]
> Need to write some optimization for our code and refactor several parts of the config. Can we create github-pages for testing the view of our component? All of our tests have been written around functions that are validating the input. We could create an overlay where we could install Vue and its' dependencies, and test our component view.
> The things should be done by the next version release:
> 
> * [ ]  Should be provided performance tests for functions
> * [ ]  Refactor of core code and using meaningful names
> * [ ]  Documentation editing
> * [ ]  Eslint config should be flexible and should control rules by APP_ENV (app environment).
> 
> @VoIdJMp, if you have any ideas, let me know and check it out (PR and the list on the top).
> 
> PS: By finishing the review, don't merge it. I will do it by myself :))

**Review by VoIdJMp — COMMENTED** [review]
fine

**Pull request merged**